docs/api_v2/api_v2_reference.apib

```diff
@@ -2918,238 +3123,413 @@ The user account is identified by the provided authentication token.
 - status:           `SUCCEEDED`                (string, required)
-- meta                                         (object, required)
-    - source:       `WEB`                      (string, required)
+- startedAt:        `2019-11-30T07:34:24.202Z` (string, required)
+- finishedAt:       `2019-12-12T09:30:12.202Z` (string, required)
+- meta                                         (object)
+    - origin:       `WEB`                      (string, required) # TODO: review
     - clientIp:     `172.234.12.34`            (string, required)
-    - userAgent:    `Mozilla/5.0 (iPad)`       (string, required)
+    - userAgent:    `Mozilla/5.0 (Macintosh)`  (string, required) # TODO: review
 
 ## BuildShort2 (object)
 - id:              `HG7ML7M8z78YcAPEB`         (string, required)
-- startedAt:        `2015-12-12T07:34:14.202Z` (string, required)
-- finishedAt:       `2015-12-13T08:36:13.202Z` (string, required)
+- startedAt:        `2019-12-12T07:34:14.202Z` (string, required)
+- finishedAt:       `2019-12-13T08:36:13.202Z` (string, required)
 - status:           `FAILED`                   (string, required)
-- meta                                         (object, required)
-    - source:       `WEB`                      (string, required)
+- meta                                         (object)
+    - origin:       `API`                      (string, required) # TODO: review
     - clientIp:     `172.234.12.34`            (string, required)
-    - userAgent:    `Mozilla/5.0 (iPad)`       (string, required)
+    - userAgent     `Mozilla/5.0 (Intel)`      (string, required) # TODO: review
 
 ## BuildAborted (object)
 - id:               `HG7ML7M8z78YcAPEB`        (string, required)
-- actId:            `johndoe~my-actor`         (string, required)
+- actId:            `janedoe~my-actor`         (string, required)
 - userId:           `klmdEpoiojmdEMlk3`        (string, required)
-- startedAt:        `2015-11-30T07:34:24.202Z` (string, required)
-- finishedAt:       `2015-12-12T09:30:12.202Z` (string, nullable)
+- startedAt:        `2019-11-30T07:34:24.202Z` (string, required)
+- finishedAt:       `2019-12-12T09:30:12.202Z` (string, nullable)
 - status:           `ABORTED`                  (string, required)
 - meta                                         (object, required)
-    - source:        `WEB`                     (string, required)
+    - origin:        `WEB`                     (string, required)
     - clientIp:      `172.234.12.34`           (string, required)
     - userAgent:     `Mozilla/5.0 (iPad)`      (string, required)
 - stats                                        (object, nullable)
     - workersUsed:   1                         (number, required)
     - durationMillis 1000                      (number, required)
-- buildNumber:       `0.0.1`                   (string, required)
-- readme:            `# Magic actor\nThis actor is magic.`            (string, nullable)
+    - runTimeSecs:   5.718                     (number, required)
```

Comment:
Same here, let's wait for the discussion above ^

Reply:
yeah